<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HEXRD</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="central_widget_layout">
    <item>
     <widget class="ImageTabWidget" name="image_tab_widget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_open">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="action_open_images"/>
     <addaction name="action_open_config"/>
     <addaction name="action_open_materials"/>
    </widget>
    <widget class="QMenu" name="menu_save">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="action_save_imageseries"/>
     <addaction name="action_save_config"/>
     <addaction name="action_save_materials"/>
    </widget>
    <addaction name="menu_open"/>
    <addaction name="menu_save"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_tabbed_view"/>
    <addaction name="action_show_toolbar"/>
    <addaction name="action_show_live_updates"/>
    <addaction name="action_show_saturation_percentages"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_edit_ims"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QDockWidget" name="config_dock_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>260</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Configuration</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="config_dock_widgets">
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolBox" name="config_tool_box">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="materials_page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>532</width>
          <height>626</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Materials</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="instrument_calibration_page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>532</width>
          <height>626</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Instrument Calibration</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTabWidget" name="calibration_tab_widget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="form_view">
            <attribute name="title">
             <string>Form View</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tree_view">
            <attribute name="title">
             <string>Tree View</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="image_view">
       <property name="text">
        <string>Image View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cartesian_view">
       <property name="text">
        <string>Cartesian View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="polar_view">
       <property name="text">
        <string>Polar View</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="color_map_dock_widget">
   <property name="windowTitle">
    <string>Color Map</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="color_map_dock_widgets">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="resolution_dock_widget">
   <property name="windowTitle">
    <string>Calibration Resolution</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="resolution_dock_widgets">
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
  </widget>
  <widget class="QDockWidget" name="frame_aggregation_widget">
   <property name="windowTitle">
    <string>Frame Aggregation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="frame_aggregation_widgets">
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
  </widget>
  <action name="action_tabbed_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tab Images</string>
   </property>
   <property name="toolTip">
    <string>View images in tabs</string>
   </property>
  </action>
  <action name="action_open_images">
   <property name="text">
    <string>Images</string>
   </property>
  </action>
  <action name="action_open_config">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="action_save_imageseries">
   <property name="text">
    <string>ImageSeries</string>
   </property>
  </action>
  <action name="action_open_materials">
   <property name="text">
    <string>Materials</string>
   </property>
  </action>
  <action name="action_save_config">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="action_show_toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
  </action>
  <action name="actionShow_Live_Updates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Live Updates</string>
   </property>
  </action>
  <action name="action_show_live_updates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Live Updates</string>
   </property>
  </action>
  <action name="action_show_saturation_percentages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Saturation Percentages</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the saturation percentages of the images when in &amp;quot;Image View&amp;quot; mode. The saturation percentages will appear as white text in the bottom left corners of the images.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_edit_ims">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Process Image Series</string>
   </property>
  </action>
  <action name="action_save_materials">
   <property name="text">
    <string>Materials</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageTabWidget</class>
   <extends>QTabWidget</extends>
   <header>image_tab_widget.py</header>
   <container>1</container>
   <slots>
    <slot>open_files()</slot>
    <slot>set_tabbed_view(bool)</slot>
    <slot>show_toolbar(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_tabbed_view</sender>
   <signal>toggled(bool)</signal>
   <receiver>image_tab_widget</receiver>
   <slot>set_tabbed_view(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_show_toolbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>image_tab_widget</receiver>
   <slot>show_toolbar(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1075</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
